e2e/errors.go

```diff
@@ -43,3 +43,20 @@ func isRetryableAPIError(err error) bool {
 
 	return false
 }
+
+// isAlreadyExistsCLIError checks for already exists error from kubectl CLI.
+func isAlreadyExistsCLIError(err error) bool {
+	if err == nil {
+		return false
+	}
+	// if multiple resources already exists. each error is separated by newline
+	errStr := strings.Split(err.Error(), "/n")
+	for _, s := range errStr {
+		// Resource already exists error message
+		if !strings.Contains(s, "Error from server (AlreadyExists)") {
```

Comment:
I do not think this is correct, the error returned by `kubectl` contains some notes, `stdout` and `stderr`, there will always be lines that do not match this.

Reply:
will fix this one. Yes, this won't work in all cases.